content/millennium/faq/common-issues.md

```diff
@@ -0,0 +1,425 @@
+Table of Contents is displayed on the left side of the window and allows
+you to quickly navigate to any area of the patient's chart. Clicking a
+PowerChart component in the Table of Contents, such as a SMART app,
+opens that component's Workspace view. The Workspace view displays and
+changes based on which PowerChart component is selected from the Table
+of Contents. Each view can include multiple windows, that are accessed
+by clicking tabs or buttons in that view. If you open your SMART app
+from the Table of Contents, information for the patient is displayed in
+the Workspace view.
+
+If your app is best used for managing a population of patients, then it
+should be optimized to be used at the Organizer view level. If your app
+is best used in the context of a single patient, it is critical that the
+user should not be able to access Organizer level information, such as
+patient lists or other patient’s data. This could be hazardous in the
+context of the single-patient Patient Chart view because users could
+become confused as to which patient’s information they are viewing. This
+is especially critical if you are porting your app to be used within the
+SMART on FHIR framework.  We have seen a few examples where developers
+didn’t remove functionality such as patient search and we had to have
+them restart their validation process once that functionality was
+removed.
+
+### Session Bleed Issue
+
+When you are running a SMART app in multiple open charts in PowerChart,
+there is potential that the app instance will share patient context and
+show same patient’s data on two separate charts.
+ 
+The application needs to avoid session bleed issue identified with fhir-
+client.js library.  Review the changes in the latest version of the
+[fhir-client.js]( https://github.com/smart-on-fhir/client-js/tree/v0.1.10)
+and pull in an additional [project released by Cerner](https://github.com/cerner/fhir-client-cerner-additions)
+to override a setting.
+ 
+Open source fhir-client.js file uses HTML5 sessionStorage parameter.
+This causes an issue within the EMR when two instances of a SMART
+application are opened on two patients.  After discovering this issue,
+we worked through the open source community to contribute the change
+back to fhir-client.js.
+
+*Note:* This cannot be properly tested until you are beginning
+validation and we are running your app directly in PowerChart. The
+following recreation steps are for your information once you have access
+to PowerChart.
+
+To recreate the issue:
+
+1. Open the chart for Patient A and launch an app.
+2. Go back to the Patient List and open an additional chart (Patient B),
+   then launch an app for that patient.
+3. On Patient A’s chart, click As Of in the upper right corner to
+   refresh the content of PowerChart. If the application displays the
+   information for Patient B within the chart for Patient A, then you
+   need to implement the change mentioned above.
+
+## Security
```

Comment:
May want a short intro as to why this is included. EG: "During application validation, we do a security review. We've gathered some common topics that are covered which should help you think through the security of your application. There are also examples of tools that can guide secure development of your application."